membership-attribute-service/test/services/AttributesMakerTest.scala

```diff
@@ -1,165 +1,240 @@
+      val result = AttributesMaker.zuoraAttributes(identityId, List(CustomerAccount(accountSummaryWithZeroBalance, Some(expiredMembership)), CustomerAccount(accountSummaryWithZeroBalance, Some(contributor))), paymentMethodResponseNoFailures, referenceDate)
+      result must be_==(expected).await
+
     }
 
     "return attributes when one sub is a recurring contribution" in {
       val expected = Some(ZuoraAttributes(
-        UserId = testId,
+        UserId = identityId,
         Tier = None,
         RecurringContributionPaymentPlan = Some("Monthly Contribution"),
         MembershipJoinDate = None
       )
       )
-      AttributesMaker.zuoraAttributes(testId, List(contributor), referenceDate) === expected
+      val result = AttributesMaker.zuoraAttributes(identityId, List(CustomerAccount(accountSummaryWithZeroBalance, Some(contributor))), paymentMethodResponseNoFailures, referenceDate)
+      result must be_==(expected).await
+
     }
 
     "return attributes relevant to both when one sub is a contribution and the other a membership" in {
       val expected = Some(ZuoraAttributes(
-        UserId = testId,
+        UserId = identityId,
         Tier = Some("Supporter"),
         RecurringContributionPaymentPlan = Some("Monthly Contribution"),
         MembershipJoinDate = Some(referenceDate)
       )
       )
-      AttributesMaker.zuoraAttributes(testId, List(contributor, membership), referenceDate) === expected
+      val result = AttributesMaker.zuoraAttributes(identityId, List(CustomerAccount(accountSummaryWithZeroBalance, Some(contributor)), CustomerAccount(accountSummaryWithZeroBalance, Some(membership))),  paymentMethodResponseNoFailures, referenceDate)
+      result must be_==(expected).await
     }
 
     "return attributes when the membership is a friend tier" in {
       val expected = Some(ZuoraAttributes(
-        UserId = testId,
+        UserId = identityId,
         Tier = Some("Friend"),
         RecurringContributionPaymentPlan = None,
         MembershipJoinDate = Some(referenceDate)
       )
       )
-      AttributesMaker.zuoraAttributes(testId, List(friend), referenceDate) === expected
+      val result = AttributesMaker.zuoraAttributes(identityId, List(CustomerAccount(accountSummaryWithZeroBalance, Some(friend))), paymentMethodResponseNoFailures, referenceDate)
+      result must be_==(expected).await
     }
 
-    "attributes" should  {
-      "return up to date Zuora attributes when they match the dynamo attributes" in {
-        val zuoraAttributes = ZuoraAttributes(
-          UserId = testId,
-          Tier = None,
-          RecurringContributionPaymentPlan = Some("Monthly Contribution"),
-          MembershipJoinDate = None,
-          DigitalSubscriptionExpiryDate = None
-        )
+    // We are currently returning actionAvailableFor = None always in AttributesMaker and just logging the value we've calculated
+    //This test should be uncommented once we resume returning the calculated value
```

Comment:
I think you can use an @Ignore annotation to have the test there (and compiled) but reported as skipped

Reply:
You've inspired me to finally look up how to do this in specs2 tests 🙂